boilerplate/bin/downloadExpoApp.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash -e
 
-# taken from https://github.com/expo/with-detox-tests/blob/master/setup.sh
+# based on https://github.com/yaron1m/expo-detox-typescript-example/blob/master/scripts/dl_expo_bins
+APP_PATH=./bin/Exponent.app
 
-# query expo.io to find most recent ipaUrl
-IPA_URL=`curl --silent --show-error https://expo.io/--/api/v2/versions |  python -c 'import sys, json; print json.load(sys.stdin)["iosUrl"]'`
+if [ ! -d $APP_PATH ]; then
```

Comment:
Let's add an `echo` to indicate what we're doing, something like this:

```
echo "First time running Detox -- downloading Expo client app"
```

Reply:
👍

Reply:
✅